scripts/packages/chocolatey/bazel.nuspec.template

```diff
@@ -0,0 +1,87 @@
+    <version>$tvVersion</version>
+    <!-- <packageSourceUrl>Where is this Chocolatey package located (think GitHub)? packageSourceUrl is highly recommended for the community feed</packageSourceUrl>-->
+    <packageSourceUrl>https://github.com/bazelbuild/bazel/tree/master/scripts/packages/chocolatey</packageSourceUrl>
+    <!-- owners is a poor name for maintainers of the package. It sticks around by this name for compatibility reasons. It basically means you. -->
+    <owners>Peter Mounce, TODO</owners>
+
+    <!-- ============================== -->
+
+    <!-- == SOFTWARE SPECIFIC SECTION == -->
+    <!-- This section is about the software itself -->
+    <title>bazel (Install)</title>
+    <authors>TODO</authors>
+    <!-- projectUrl is required for the community feed -->
+    <projectUrl>https://bazel.io</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/bazelbuild/bazel/master/site/images/bazel-logo.png</iconUrl>
+    <copyright>Copyright 2016 The Bazel Authors. All rights reserved.</copyright>
+    <!-- If there is a license Url available, it is is required for the community feed -->
+    <licenseUrl>https://github.com/bazelbuild/bazel/tree/master/LICENSE.txt</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/bazelbuild/bazel.git</projectSourceUrl>
+    <docsUrl>https://www.bazel.io/versions/master/docs/getting-started.html</docsUrl>
+    <mailingListUrl>https://groups.google.com/forum/#!forum/bazel-discuss</mailingListUrl>
+    <bugTrackerUrl>https://github.com/bazelbuild/bazel/issues</bugTrackerUrl>
+    <tags>bazel build automation</tags>
+    <summary>Correct, reproducible, fast builds for everyone</summary>
+    <description>Build software of any size, quickly and reliably, just as engineers do at Google.
+
+Bazel is Google's own build tool, now publicly available in Beta.
+
+Bazel has built-in support for building both client and server software, including client applications for both Android and iOS platforms. It also provides an extensible framework that you can use to develop your own build rules.
+</description>
+    <releaseNotes>https://github.com/bazelbuild/bazel/releases/tag/$tvVersion</releaseNotes>
+    <!-- =============================== -->      
+
+    <!-- Specifying dependencies and version ranges? https://docs.nuget.org/create/versioning#specifying-version-ranges-in-.nuspec-files -->
+    <!--<dependencies>
+      <dependency id="" version="__MINIMUM_VERSION__" />
+      <dependency id="" version="[__EXACT_VERSION__]" />
+      <dependency id="" version="[_MIN_VERSION_INCLUSIVE, MAX_VERSION_INCLUSIVE]" />
+      <dependency id="" version="[_MIN_VERSION_INCLUSIVE, MAX_VERSION_EXCLUSIVE)" />
+      <dependency id="" />
+      <dependency id="chocolatey-uninstall.extension" />
+    </dependencies>-->
+    <dependencies>
+      <dependency id="jdk8" version="[8.0.102,)"/>
+      <dependency id="msys2" version="[20150916.0.1,)"/>
```

Comment:
@dslomov here is where I have declared a dependency on the msys2 package. I have another PR open to that package to update its contents from 20150916 to 2016latest (userzimmermann/choco-packages#4).

[These are the nuspec docs for dependencies](https://docs.nuget.org/create/nuspec-reference#specifying-dependencies) - I have chosen to use the latest available versions of the dependencies that I'm aware of bazel having.